stackslib/src/net/httpcore.rs

```diff
@@ -247,6 +247,17 @@ pub mod request {
         Ok(txid)
     }
 
+    /// Get a clarity key (MARF or Metadata) from a path's captures, given the name of the regex field.
+    pub fn get_clarity_key(captures: &Captures, clarity_key: &str) -> Result<String, HttpError> {
```

Comment:
Because this is specific to your handler, you should implement it there instead.